tests/data/packages/teaser_single/package.mo

```diff
@@ -2,5 +2,5 @@ package teaser_single
   extends Modelica.Icons.Package;
 
   annotation (uses(
-    Modelica(version="4.0.0"), Buildings(version="10.0.0")), version="1");
+    Modelica(version="4.0.0"), Buildings(version="11.0.0")), version="1");
```

Comment:
we should figure out how to make this a variable. We don't need to do it now, but this should be set from somewhere else Modelica builder shouldn't care about the version of MBL/Buildings.